### DISCUSSION TITLE
Navidrome Theme

### DISCUSSION THREAD

**Discussion by bwest2102** [navidrome/navidrome#3451] [Q&A]
When I apply a theme to Navidrome and come back on the same device, the theme is set back to the original theme. Is there an environment variable I can set to keep the change permanent? It's not a big deal if not.

**Comment by deluan** [comment]
Yes, `DefaultTheme` or `ND_DEFAULTTHEME`: https://www.navidrome.org/docs/usage/configuration-options/#:~:text=%22en%22-,DefaultTheme,-ND_DEFAULTTHEME

But it saves the theme in the browser's localStorage, so it should keep the last theme selected.
